src/gpl/src/routeBase.cpp

```diff
@@ -675,11 +675,14 @@ std::pair<bool, bool> RouteBase::routability()
   }
 
   dbBlock* block = db_->getChip()->getBlock();
-  log_->info(GPL,
-             45,
-             "{:20} {:10.3f} um^2",
-             "InflatedAreaDelta:",
-             block->dbuAreaToMicrons(inflatedAreaDelta_));
+  log_->info(
+      GPL,
+      45,
+      "{:20} {:10.3f} um^2 ({:+.2f}%)",
+      "InflatedAreaDelta:",
+      block->dbuAreaToMicrons(inflatedAreaDelta_),
+      (static_cast<float>(inflatedAreaDelta_) / nbVec_[0]->nesterovInstsArea())
+          * 100.0f);
```

Comment:
This block is very confuse. Please refactor lines 684 and 685 to a variable with meaningful name.

Also, it seems that the `InflatedAreaDelta` string is an argument to the fmt string below? Maybe simplify it to `"InflatedAreaDelta: {:10.3f} um^2 ({:+.2f}%)",`

Reply:
This format `"{:20} {:10.3f} um^2 ({:+.2f}%)",` is easier to align with other messages, for example:

![image](https://github.com/user-attachments/assets/775f2618-8073-435c-83ca-d1a0a5cbb08a)

Reply:
Makes sense, thanks!